src/main/resources/default/templates/biz/tycho/analytics/data-explorer.html.pasta

```diff
@@ -0,0 +1,466 @@
+                                    data-range="lastYear"
+                                    onclick="changeRange('lastYear')">
+                                @i18n("DataExplorerController.lastYear")
+                            </button>
+                        </div>
+                    </div>
+                    <div class="col-12 col-md-6 mb-2">
+                        <div class="mb-2 text-small text-muted">@i18n("DataExplorerController.comparisonPeriod")</div>
+                        <div class="btn-group">
+                            <button type="button"
+                                    class="period-select-js btn btn-outline-primary"
+                                    data-period="none"
+                                    onclick="changePeriod('none')">@i18n("DataExplorerController.periodOff")
+                            </button>
+                            <button type="button"
+                                    class="period-select-js btn btn-primary"
+                                    data-period="lastYear"
+                                    onclick="changePeriod('lastYear')">@i18n("DataExplorerController.periodLastYear")
+                            </button>
+                            <button type="button"
+                                    class="period-select-js btn btn-outline-primary"
+                                    data-period="lastMonth"
+                                    onclick="changePeriod('lastMonth')">@i18n("DataExplorerController.periodLastMonth")
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="charts">
+
+        </div>
+    </div>
+
+    <script type="text/javascript">
+        let range = 'last12Months';
+        let comparisonPeriod = 'lastYear';
+
+        // Maps typs of charts to their actual JS callbacks...
+        const CHART_TYPES = {
+            'line': lineChartType,
+            'area': areaChartType
+        };
+
+        // Tries to find the chart DIV with the given identifier (TYPE or TYPE:OBJECT).
+        function findChart(identifier) {
+            return document.querySelector('.chart-js[data-identifier=\'' + identifier + '\']');
+        }
+
+        // Adds or updates the chart for the given identifier.
+        function addChart(identifier) {
+            $('#select-entity-modal').modal('hide');
+
+            const _existingChart = findChart(identifier);
+            if (_existingChart) {
+                hideTypeSelector();
+
+                return recompute(_existingChart);
+            }
+
+            const _charts = document.getElementById("charts");
+            const _chart = document.createElement('DIV');
+            _chart.dataset.identifier = identifier;
+            _chart.classList.add('chart-js');
+            _chart.classList.add('card');
+            _chart.classList.add('mb-4');
```

Comment:
Small tip: It is somewhat shorter to do the following and should also work:
`_chart.classList.add('chart-js', 'card', 'mb-4')`